src/enclave/rpc_sessions.h

```diff
@@ -192,7 +192,8 @@ namespace enclave
       // Caller authentication is done by each frontend by looking up
       // the caller's certificate in the relevant store table. The caller
       // certificate does not have to be signed by a known CA (nullptr).
-      cert = std::make_shared<tls::Cert>(nullptr, cert_, pk, tls::auth_default);
+      cert =
+        std::make_shared<tls::Cert>(nullptr, cert_, pk, std::nullopt, false);
```

Comment:
@jumaffre: I think you were the last one to touch this piece of code. This used to require `auth_optional`, which does not exist anymore. Do you have any concerns about the certificate validation being disabled/ignored here? (I think it's fine, just a sanity check.)

Reply:
LGTM. Perhaps mention why `auth_required` is `false` too in the comment above?